### ISSUE TITLE
File with backwards title shows weird results

### ISSUE THREAD

**Issue by OmgImAlexis** [guessit-io/guessit#394]
At times I get nzb files with a backwards title but a correct title for the dir and this causes some weirdness in guessit.

```
# guessit: 2.1.2.dev0  rebulk: 0.8.3.dev0
? The.Messengers.2015.S01E07.1080p.WEB-DL.DD5.1.H264.Nlsubs-Q/QoQ-sbuSLN.462.H.1.5DD.LD-BEW.p0801.70E10S.5102.sregnesseM.ehT.mkv
: title: The Messengers
  year: 2015
  screen_size: 1080p
  format: WEB-DL
  audio_codec: AC3
  audio_channels: 5.1
  video_codec: h264
  subtitle_language: nl
  release_group: Q
  episode: [1, 2]
  other: LD
  season: 51
  container: mkv
  mimetype: video/x-matroska
  type: episode
  parsing_time: 0.226768016815
```

**Comment by OmgImAlexis** [comment]
Looks like it's mainly [QoQ who do this](https://github.com/clinton-hall/nzbToMedia/issues/385).

**Comment by OmgImAlexis** [comment]
For: /Finding.Carter.S02E01.Love.the.Way.You.Lie.1080p.WEB-DL.AAC2.0.H.264-NL/LN-462.H.0.2CAA.LD-BEW.p0801.eiL.uoY.yaW.eht.evoL.10E20S.retraC.gnidniF.mkv
GuessIt found: {
    "title": "Finding Carter", 
    "episode_title": "Love the Way You Lie", 
    "screen_size": "1080p", 
    "format": "WEB-DL", 
    "audio_codec": "AAC", 
    "audio_channels": "2.0", 
    "video_codec": "h264", 
    "language": "Dutch", 
    "season": 4, 
    "episode": 62, 
    "other": "LD", 
    "release_group": "BEW.p0801.eiL.uoY.yaW.eht.evoL.10E20S.retraC.gnidniF", 
    "container": "mkv", 
    "mimetype": "video/x-matroska", 
    "type": "episode"
}

**Comment by ratoaq2** [comment]
Fixed by #405
